packages/ts/react-crud/src/autocrud.obj.css

```diff
@@ -8,6 +8,22 @@
   flex-direction: column;
 }
 
+.auto-crud-actions {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  padding: 0 var(--lumo-space-xs);
+  display: flex;
+  gap: var(--lumo-space-xs);
+  align-items: center;
+}
+
+.auto-crud-actions vaadin-button {
+  margin: 0;
+}
```

Comment:
Elaborate styles to minimize the padding, and thus the width, of the column which has a tricky to customize padding otherwise.